shopinvader_search_engine_update_pricelist/models/product_pricelist.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2024 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import  models
+
+class PricelistItem(models.Model):
+    _name = "product.pricelist.item"
+    _inherit = ["product.pricelist.item", "se.product.update.mixin"]
+
+    def get_products(self):
+        Product = self.env['product.product']
+        return Product.search(self._get_product_domain())
+    
+    def _get_product_domain(self):
```

Comment:
```suggestion
    def _se_get_product_domain(self):
```
and I would suggest to rename the methods of the mixin to be more explicit about their scope (eg: `get_products` seems quite generic.